src/bundle/Resources/config/serializer.yaml

```diff
@@ -35,5 +27,13 @@ services:
 
     ibexa.rest.serializer.object_normalizer:
         class: Symfony\Component\Serializer\Normalizer\ObjectNormalizer
+        tags:
+            - { name: ibexa.rest.serializer.normalizer, priority: -999 }
+
+    Ibexa\Contracts\Rest\Output\VisitorAdapterNormalizer:

```

Comment:
Any reason why you've moved declaration? This will make conflict resolution later slightly more... annoying :)

Reply:
Ah, simply because I wanted to order it by priority, I can move it back if you prefer it like that

Reply:
Moved it back

Reply:
Didn't you want to set a different priority though? :)

Reply:
It didn't work with different priority but I've changed `ObjectNormalizer`'s priority, VisitorAdapterNormalizer was always `-1000`

Reply:
So the change was not intentional, and both services are expected to have `-1000` as priority - as is in the current version of PR? It's not a mistake?